az-hadoop-jobtype-plugin/src/main/java/azkaban/jobtype/HadoopJobUtils.java

```diff
@@ -593,4 +598,21 @@ public static String javaOptStringFromHadoopConfiguration(Configuration conf, St
     }
     return String.format("-D%s=%s", key, value);
   }
+
+  /**
+   * Construct a CSV of tags for the Hadoop application.
+   *
+   * @param List of keys to construct tags from.
+   * @return a CSV of tags
+   */
+  public static String constructHadoopTags(Props props, String[] keys) {
```

Comment:
A quick question. will this method be used by Spark job? If not, we may not use public for that? Also, that would be great if we can have some basic tests for this function.

Reply:
Thanks for the review @kunkun-tang, yes it will be used by spark. I added some unit tests for this function